scripts/travis_yml_test.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test to check that travis.yml file & protractor.conf.js have the
+same test suites"""
```

Comment:
End docstrings with periods. (In general, write full sentences.)

Put an empty newline below this one, too.

Reply:
Done.